ui/widgets/widgets-dir/strapi-config-app/.env

```diff
@@ -11,11 +11,11 @@ PORT=3001
 # REACT_APP_PUBLIC_URL=http://172.40.0.142:8081/api/config/
 
 ######------- Local ENV -----------
-# REACT_APP_KEYCLOAK_URL=http://20.79.198.96.nip.io/auth
```

Comment:
remove commented code if not used. If necessary, move this to a different .env file